cluster_traffic_capture/demo_haproxy.py

```diff
@@ -159,6 +191,12 @@ def subshell_print(message: str):
         ports=[dfc.PortMapping(HAPROXY_INTERNAL_PORT, HAPROXY_PRIMARY_PORT)],
         volumes=[],
         ulimits=[Ulimit(name='memlock', soft=-1, hard=-1)],
+        env_kv={
+            "AWS_REGION": aws_region,
+            "AWS_ACCESS_KEY_ID": aws_access_key_id,
+            "AWS_SECRET_ACCESS_KEY": aws_secret_access_key,
+            "AWS_SESSION_TOKEN": aws_session_token
+        },
```

Comment:
I'm not sure I have a great understanding of when you use the `env_kv` vs the passthrough -- why are this in the `env_kv` instead of passthrough?

Reply:
The current Docker SDK we're using doesn't invoke CLI commands via the shell and so it can't do passthrough unfortunately.  Therefore, we need to use the K/V approach.  I left the passthrough code I wrote in the PR because I think the path forward is to move away from the Python Docker SDK and just invoke the CLI manually using our own code.